boards/sparkfun/artemis_nano/board.cmake

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+if(BOARD STREQUAL "artemis_nano")
```

Comment:
use the proper board Kconfig symbol

Reply:
Sorry I did not get your command on this. Could you please explain it a bit?

Reply:
`if(CONFIG_BOARD_ARTEMIS_NANO)` would be used, but you can remove it entirely since board.cmake is only used if this board is used, so unless this board has multiple board targets the if can go